dashboard/app/controllers/admin_reports_controller.rb

```diff
@@ -40,29 +43,40 @@ def diversity_survey
         next if survey_result.properties.blank?
         @respondents += 1
 
+        # The number of born students associated with the teacher.
```

Comment:
The user has signed into their account (note that [very] roughly a third of our accounts are unborn.).